### DISCUSSION TITLE
身健在，且加班 | /cn/2019/04/996-icu/

### DISCUSSION THREAD

**Discussion by yihui** [yihui/yihui.org#1325] [Comments]
https://yihui.org/cn/2019/04/996-icu/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
《干法》。

好久不见，也好久没见你冒泡！

> ——原帖发布于 2019-09-12 14:07:44

**Reply by yihui** [reply]
上次冒泡七年前：<https://yihui.name/cn/2012/06/quick-notes/#comment-595659432>

你跟这位读者很像，几乎说了跟他一样的话：<https://yihui.name/cn/2018/08/pagination/#comment-4074608028>

> ——原帖发布于 2019-09-12 15:50:06

[a comment by an automated account is not shown]

**Comment by yongchuang1625** [comment]
我在想，居家办公不会很难专注工作吗 比如家里美食、电视、游戏等各种的诱惑，以及家中杂事的干扰

**Reply by yihui** [reply]
家里没什么美食（美食对我诱惑力也不大），我也几乎不看电视（只在电脑上看羽毛球视频）、几乎从不打游戏（缺这根筋、没兴趣）。至于杂事干扰，其实主要就是小朋友；只要他们上学去了，就万事大吉，没什么别的杂事。
